perf/bench/coinbase/prep

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh
+#
+# Fetch rules and targets prior for the "r2c" benchmark.
+#
+# rule_dir: input/rules
+# target_dir: input/bifrost
+#
+# Uses sh because bash is not installed within the semgrep docker container.
+#
+set -eu
+
+mkdir -p input
+mkdir -p input/rules
+
+cp r2c-ci.yml input/rules
```

Comment:
again, can you include the command you used to generate those files,
and I would just cat them together and generate a single file instead of putting in input/rules/
to be more consistent with the other corpus which use a single .yml file (which is also useful for semgrep-core).